back/engines/commercial/project_folders/app/models/project_folders/folder.rb

```diff
@@ -45,6 +47,10 @@ class Folder < ::ApplicationRecord
 
     after_destroy :remove_moderators
 
+    pg_search_scope :search_by_all,
+      against: %i[title_multiloc description_multiloc description_preview_multiloc slug],
```

Comment:
Maybe it makes sense to add `slug` to `models/project.rb` too?

Reply:
Agree. Makes sense to be consistent one way or the other.